### PR TITLE
pam-krb5: update to 4.10.

### PR DESCRIPTION
**Pull request by ailiop-git**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by ericonr** [comment]
@Vaelatern

**Comment by Vaelatern** [comment]
I see a bunch of failed tests. I'd hope a bump considers them. I no longer use this package.

**Comment by paper42** [comment]
ping @ailiop-git

File: srcpkgs/pam-krb5/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'pam-krb5'
 pkgname=pam-krb5
-version=4.9
-revision=2
+version=4.10
```
Comment (ericonr) [inline srcpkgs/pam-krb5/template:3]: 4.11 is out

**Pull request closed without merging**